glean-core/Cargo.toml

```diff
@@ -42,7 +42,7 @@ crossbeam-channel = "0.5"
 thiserror = "1.0.4"
 uniffi = { version = "0.23.0", default-features = false }
 time = "0.1.40"
-remove_dir_all = "0.5.3"
+remove_dir_all = "0.8.1"
```

Comment:
The current failing on CI seems to indicate that this version bump requires a specific version bump of rustc. Before merging this, we should make sure what's the minimum required version on m-c. Leaving r- for now until we figure this out.

Reply:
Looks like `rust-version` is pinned on lots of different versions in m-c.  From very early versions up to a few newer versions tha 1.62 that we would need.

Application-Services doesn't pin any versions that I can see.